src/utilities/generatePassword.ts

```
@@ -0,0 +1,15 @@
+export function generatePassword(length: number = 12): string {
+  const chars =
+    'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*()_+[]{}|;:,.<>?'
+  let password = ''
+  const randomValues = new Uint32Array(length)
+  if (crypto && crypto.getRandomValues) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)